store/tikv/2pc.go

```diff
@@ -130,7 +160,9 @@ func newTwoPhaseCommitter(txn *tikvTxn, connID uint64) (*twoPhaseCommitter, erro
 			}
 			delCnt++
 		}
-		keys = append(keys, k)
+		if isPessimistic && !bytes.Equal(k, c.primaryKey) {
```

Comment:
maybe should be:
```suggestion
		if !isPessimistic || !bytes.Equal(k, c.primaryKey) {
```

Reply:
after dig, maybe old one is ok too, c.primaryKey always be null if `isPessimistic=true`

Reply:
ping @coocood

Reply:
@lysu I know, but add `isPessimistic` is more explicit that this code is only related to pessimistic lock.